setup.py

```diff
@@ -23,7 +23,7 @@ def get_target_version():
     ),
     packages=setuptools.find_packages(),
     include_package_data=True,
-    install_requires=["requests", "pandas", "docker", "jinja2", "cryptography"],
+    install_requires=["requests", "pandas", "docker", "jinja2", "cryptography", "torch", "transformers", "onnxruntime"],
```

Comment:
@lesters @oyving 

I feel it is overkill to add those three libraries as dependencies to pyvespa. 

* `torch` and `transformers` are required if the user wants to build applications with BERT model.
* `onnxruntime` is only used for a unit test.

Do you think that implementing something like [this](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_graph_to_onnx.py#L267-L271) is more sensible than requiring everyone to install all those libraries?

Reply:
I think that is reasonable, but these are requirements for passing the suit of unit tests run in the build process?

Reply:
Yes, but I can have a requirements.txt file responsible for installing the optional packages during the CI pipeline.

Reply:
Yeah, I think that makes sense. Make the dependencies marked as extras in the file and use requirements.txt to make sure everything is in place during testing.

You should also make sure we have tests that test the behavior if the dependencies are absent.

Reply:
I guess you are going to follow this guide? https://setuptools.readthedocs.io/en/latest/userguide/dependency_management.html#optional-dependencies

Reply:
That does sound sensible. +1